pom.xml

```diff
@@ -6,7 +6,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.iyzipay</groupId>
     <artifactId>iyzipay-java</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.6</version>
```

Comment:
The versions increased by continuous delivery server. You don't need to set the version.